store/store.go

```diff
@@ -415,11 +415,27 @@ func (s *store) IsBanned(addr crypto.Address) bool {
 	return s.config.BannedAddrs[addr]
 }
 
-// IsPruned indicates that store is in prune mode.
+// IsPruned returns true if the store is in prune mode, otherwise false.
 func (s *store) IsPruned() bool {
 	return s.isPruned
 }
 
+// PruningHeight returns the height at which blocks will be pruned if the store is in prune mode.
+// If the store is not in prune mode, it returns 0.
+func (s *store) PruningHeight() uint32 {
+	s.lk.RLock()
+	defer s.lk.RUnlock()
+
+	if !s.isPruned {
+		return 0
+	}
+
+	// TODO: it can be optimized (and safer?) by keeping the last block height in memory.
+	cert := s.lastCertificate()
```

Comment:
@b00f We can check if cert is nil here.

Reply:
This is not efficient for check, is prune one time set for prune state.

Reply:
Thanks, got it.